backend/src/main/java/util/ThreadHelper.java

```diff
@@ -0,0 +1,18 @@
+package util;
+
+import java.time.Duration;
+import java.util.logging.Logger;
+
+public class ThreadHelper {
+    private static final Logger logger = LogHelper.getLogger(ThreadHelper.class.getName());
+
+    public static void snooze(Duration duration) {
+        try {
+            java.lang.Thread.sleep(duration.toMillis());
```

Comment:
Wouldn't `Thread.sleep(duration.toMillis())` work as well? I don't see where the `Thread` below is coming from.

Reply:
Fixed! No idea how this sneaked in here... ?
Thread is part of java.lang, so no import needed.